dashboard/app/controllers/api_controller.rb

```diff
@@ -38,6 +37,32 @@ def google_classrooms
     end
   end
 
+  def google_classrooms
+    query_google_classroom_service do |service|
+      response = service.list_courses(teacher_id: 'me')
+      render json: response.to_h
+    end
+  end
+
+  def import_google_classroom
+    course_id = params[:course_id].to_s
+
+    query_google_classroom_service do |service|
+      students = service.list_course_students(course_id).students.map do |student|
+        OmniAuth::AuthHash.new(
+          uid: student.user_id,
+          provider: 'google_oauth2',
+          info: {
+            name: student.profile.name.full_name
+          },
+        )
+      end
+
+      section = Section.from_omniauth(course_id, Section::LOGIN_TYPE_GOOGLE_CLASSROOM, students, current_user.id)
```

Comment:
I think hidden section codes like `G-6893559139` are a good option.  I'm tempted to create a new `GoogleClassroomSection < Section` that abstracts away some of this.  Thoughts?